.github/workflows/python-ci.yml

```diff
@@ -34,12 +34,12 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python-version: [ '3.6', '3.9', '3.10' ]
+        python-version: [ '3.8', '3.9', '3.10' ]
         django-version: [ '3.2.*', '4.1.*' ]
         exclude:
-          - python-version: '3.9'  # no need to test with 3.9, already tested with 3.6 and 3.10
+          - python-version: '3.9'  # no need to test with 3.9, already tested with 3.8 and 3.10
             django-version: '3.2.*'
-          - python-version: '3.6'
+          - python-version: '3.8'
```

Comment:
you should not just adapt, you should fix. keep 3.8 and 3.10 and delete exclude section